feature_engine/categorical_encoders.py

```diff
@@ -689,6 +689,15 @@ class RareLabelCategoricalEncoder(BaseEstimator, TransformerMixin):
         to find frequent labels. If the variable contains less categories, all
         of them will be considered frequent.
 
+    max_n_categories: int, default=None
+        the maximum no of categories that should be considered frequent.
+        If None, all categories with frequency above the tolerance will be
+        considered.
+
+        .. note::
```

Comment:
Hi @SuryaThiru I think this 3 lines in the docs are not needed, since you well pointed out that the desired behaviour is to check for the max categories that fullfill the tol criteria.